src/testing_system/internal/usecase/Runner.py

```
@@ -72,7 +70,7 @@ async def _process_worker(self, executor):
                 self.experiment.add_answer(processed)
                 await self.eval_queue.put((query,processed))
             except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)